test/jdk/java/util/Formatter/Basic-X.java.template

```diff
@@ -931,6 +931,16 @@ public class Basic$Type$ extends Basic {
         test("%3.0e", "1e+07",  10000000.00);
         test("%3.0e", "1e+08", 100000000.00);
 
+        //---------------------------------------------------------------------
+        // %e - adoption of Double.toString(double) algorithm (8300869)
```

Comment:
Nit: we generally don't use the bug ids in comments like this, usually at most relying on a bug listed in the test "@bug" line.

Reply:
The bug id was added there because I'm not sure there's a way to add the @bug line to Basic-X.java.template`.

Reply:
I see; it might suffice in this case to rely on the bug information in the SCM history, but I don't have a strong opinion.

Reply:
In this specific `*.java.template` case, I think that the bug id in the comment somehow replaces the lack of the @bug line.
But I have no problems in removing it if there are stronger opinions.

Reply:
The `@bug` tag is in `BasicTestLauncher` which is where 8300869 should likely go.

Reply:
Thanks @bplb for pointing to the right place.
Done